eZ/Publish/Core/FieldType/DateAndTime/SearchField.php

```diff
@@ -76,4 +77,32 @@ public function getDefaultSortField()
     {
         return $this->getDefaultMatchField();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFullTextData(Field $field, FieldDefinition $fieldDefinition)
+    {
+        $dateTime = new DateTime();
+        $dateTime->setTimestamp($field->value->data['timestamp']);
+
+        return [
+            // format taken from {@see \eZ\Publish\Core\FieldType\DateAndTime\Value::$stringFormat}
+            $dateTime->format('U'),
```

Comment:
Right, we agreed before not to change current behavior, I must've forgotten about this code. Moreover Epoch for FullText makes no sense :) Fixed it in 06e437d.